File: auto-docs/executables/nodejs/horizontal_histogram.js
var plotly = require('plotly')('TestBot', 'r1neazxo9w')
var y = [];

for (var i = 0; i < 500; i ++) {
	y[i] = Math.random();
}


var data = [
  {
    y: y, 
    type: "histogram"
  }
];
var graph_options = {fileopt: "overwrite", filename: "horizontal-histogram"}
plotly.plot(data, graph_options, function (err, msg) {
    console.log(msg);
});

File: auto-docs/executables/matlab/horizontal_histogram.m
signin('TestBot', 'r1neazxo9w')
y = randn(500,1);

data = {...
  struct(...
    'y', y, ...
    'type', 'histogram')...
};
response = plotly(data, struct('fileopt', 'overwrite', 'filename', 'horizontal-histogram'));
plot_url = response.url